server/src/main/java/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -608,7 +608,9 @@ protected Long restart(final HaWorkVO work) {
 
             VMInstanceVO started = _instanceDao.findById(vm.getId());
             if (started != null && started.getState() == VirtualMachine.State.Running) {
+                String message = String.format("HA on VM: %s", started.getHostName());
```

Comment:
I like the idea of enhancing the alerts.
What do you think of changing from `Ha on VM: VM Name` to `Ha Starting VM: VM Name (i-2-13477-VM)`.

Thus, making it `String.format("HA on VM: %s (%s)", started.getHostName(), started.getInstanceName());`.

Reply:
@ravening Do you want to add instance name to the alert message (as advised)?